docs/integrations/README.md

```diff
@@ -0,0 +1,21 @@
+# Connectors
+
+## Stage of Connectors
+
+Instill AI provides connectors for users integration, and we propose a `Stage` to users for understanding what current status are of the connectors
+
+**XXXXX**:
+
+**Beta**: The connector is well tested and is expected to work a majority of the time
+
+**Alpha**: This connector is not fully tested
+
+### Sources
+| Connector | Stage |
+| :--- | :--- |
+| Direct | Alpha |
+
+### Destinations
```

Comment:
For `Data Sources` and `Data Destinations` tables, do you think we should add a `Description` column to describe what the connector is for? Because we didn't mention anything about `Direct` in the documentation.

Reply:
Sure, I can do that

Reply:
> For `Data Sources` and `Data Destinations` tables, do you think we should add a `Description` column to describe what the connector is for? Because we didn't mention anything about `Direct` in the documentation.

I've also added the description accordingly in this [commit](https://github.com/instill-ai/vdp/pull/59/commits/316f4082f8b067e4b81114f077bc9f398e74703a)